.gitignore

```diff
@@ -1,11 +1,7 @@
-# Created by .ignore support plugin (hsz.mobi)
-### JetBrains template
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
-# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
-
 .idea/
 .gradle/
 build/
+.DS_Store
```

Comment:
would be better to add a global git ignore and then you wouldn't need to add this in all the projects you work in